tsconfig.json

```diff
@@ -10,6 +10,8 @@
       "emitDecoratorMetadata": true,
       "moduleResolution": "node",
       "importHelpers": true,
+      "lib": ["es6", "dom"],
```

Comment:
What do we need `dom` for?

Reply:
I was testing reloading the page

Reply:
Does it mean we do not need it anymore? If so remove.